cmd/tdex/depositfee.go

```diff
@@ -1,23 +1,51 @@
 package main
 
 import (
+	"bufio"
+	"bytes"
 	"context"
+	"encoding/hex"
+	"os"
+	"strings"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/tdex-network/tdex-daemon/pkg/explorer"
+	"github.com/tdex-network/tdex-daemon/pkg/explorer/esplora"
+	"github.com/tdex-network/tdex-daemon/pkg/trade"
+	"github.com/tdex-network/tdex-daemon/pkg/transactionutil"
+	"github.com/tdex-network/tdex-daemon/pkg/wallet"
 
 	pboperator "github.com/tdex-network/tdex-protobuf/generated/go/operator"
 
 	"github.com/urfave/cli/v2"
 )
 
+const (
+	MinFee          = 5000
+	MaxNumOfOutputs = 150
+)
+
 var depositfee = cli.Command{
-	Name:  "depositfee",
-	Usage: "get a deposit address for the fee account used to subsidize liquid network fees",
+	Name:   "depositfee",
+	Usage:  "get a deposit address for the fee account used to subsidize liquid network fees",
+	Action: depositFeeAction,
 	Flags: []cli.Flag{
+		&cli.BoolFlag{
+			Name:  "no_fragments",
+			Usage: "disable utxo fragmentation",
+			Value: false,
+		},
+		&cli.StringFlag{
+			Name:  "explorer",
+			Usage: "explorer endpoint url",
+			Value: "http://127.0.0.1:3001",
```

Comment:
put the liquid blockstream one as default, for regtest we will have to pass

Reply:
@tiero This flag has been removed. The explorer is always sourced from state and if it's not set via `tdex-cli config set explorer_url http://localhost:3001`, for example, it defaults to the liquid blockstream one.